app/uk/gov/hmrc/hmrcemailrenderer/templates/mods/ModsTemplates.scala

```diff
@@ -26,7 +26,13 @@ object ModsTemplates {
       templateId = "mods_import_declaration",
       fromAddress = govUkTeamAddress,
       service = MODS,
-      subject = params => s"Import declaration and payment for commercial goods - ${params.get("declarationReference")}",
+      subject = params => {
+         if(params("emailTo") == "BorderForce") {
+           s"${params("surname")} - ${params("declarationReference")} - Import declaration"
```

Comment:
we add `surname` to the subjects when we send emails to BorderForce 
for normal traders we just show the subject as shown in the below `else` block with out names